rpc-jobs/multi-node-aio.yml

```diff
@@ -45,9 +45,11 @@
       - string:
           name: STAGES
           default: |
-            Allocate Resources, Connect Slave, Prepare Multi-Node AIO, Prepare RPC Configs,
-             Deploy RPC w/ Script, Install Tempest, Tempest Tests, Prepare Horizon Selenium,
-             Horizon Tests, Prepare Kibana Selenium, Kibana Tests, Cleanup
+            Allocate Resources, Connect Slave, Prepare Multi-Node AIO,
+             Prepare RPC Configs, Deploy RPC w/ Script, Prepare MaaS Configs,
```

Comment:
There's still `Prepare MaaS Configs` instead of `Prepare Configs`, though that might be moot if we don't make Prepare Configs its own stage.